modules/ListView.js

```diff
@@ -293,8 +310,19 @@
      * @method reorder
      */
     reorder: function() {
-      var firstChildView, elements = [];
-     _.each(this.modelsToRender(), function(model, index) {
+      var firstChildView, sameOrder,
+        elements = [],
+        models = this.modelsToRender(),
+        newOrderOfIds = _.pluck(models, this.__modelId);
+      if (_.size(newOrderOfIds) === _.size(this.__orderedModelIdList)) {
```

Comment:
explicitly call out sameSize as a var.  Use it here and on line 322 instead of hiding it with sameOrder.